src/kafka/producer.js

```diff
@@ -224,7 +228,11 @@ class Producer extends EventEmitter {
 
       this._producer.on('ready', (args) => {
         logger.silly(`Native producer ready v. ${Kafka.librdkafkaVersion}, e. ${Kafka.features.join(', ')}.`)
```

Comment:
Please change this to debug.

Reply:
> Please change this to debug.

I think there is also a similar logger for the consumer, please also align that to debug.